packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js

```diff
@@ -1951,6 +1953,17 @@ function flushSyncWork() {
   }
 }
 
+function requestFormReset(form: HTMLFormElement) {
+  previousDispatcher.r(/* requestFormReset */ form);
+  const formInst = getInstanceFromNodeDOMTree(form);
+  if (formInst !== null) {
+    requestFormResetOnFiber(formInst);
+  } else {
+    // TODO: What if none of the dispatchers find a matching form instance?
+    // Should we detect this in dev and warn?
```

Comment:
that's interesting. Feels like it should be an error but then we'd need to implement that in the isomorphic dispatcher which we probably shouldn't do. we can have it return something and if no deeper dispatcher handled it we can just call form.reset() here or we can error maybe?

Reply:
Yeah I think maybe an error is the best option. It would error if you passed a non-form element into it, so that would make it consistent.

Calling `form.reset` immediately doesn't seem great, especially without a warning, because it's almost certainly not what you intended to happen.